jena-parent/pom.xml

```diff
@@ -59,10 +59,10 @@
     <ver.commonsio>2.5</ver.commonsio>
     
     <ver.commonslang3>3.4</ver.commonslang3>
-    <ver.commonscsv>1.3</ver.commonscsv>
-    <ver.dexxcollection>0.6</ver.dexxcollection>
+    <ver.commonscsv>1.4</ver.commonscsv>
+    <ver.dexxcollection>0.7</ver.dexxcollection>
 
-    <ver.httpclient>4.5.2</ver.httpclient>
+    <ver.httpclient>4.5.3</ver.httpclient>
     <ver.httpcore>4.4.4</ver.httpcore>
```

Comment:
This needs to be 4.4.6 to align with httpclient 4.5.3 (from http://central.maven.org/maven2/org/apache/httpcomponents/httpcomponents-client/4.5.3/httpcomponents-client-4.5.3.pom)